### PR TITLE
Review Selection and Focus styling adjusted

### PR DESCRIPTION
**Pull request by nielslyngsoe**
This PR changes the selection color and focus outline color. This as well involves correction the look of focus in different places.

A few screenshots showing the new look:

image

image

Focus outline adjustments in various places:
image
image
image

Focus outline adjustments, to be visible on top of selected item:

image

Also enabling keyboard-focus in the sub-header:

image
image
image

I as well took the option to align the search and create button on the Users section, to match other lists:
image

---
_This item has been added to our backlog [AB#4364](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/4364)_

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/less/variables.less [hunk 1]
```diff
@@ -108,6 +108,7 @@
 //@blueLight:               #4f89de;
 @blue:                  #2E8AEA;
 @blueMid:               #2152A3;// updated 2019
+@blueMidLight:          rgb(99, 174, 236);
```
Comment (warrenbuckley) [inline src/Umbraco.Web.UI.Client/src/less/variables.less]: ANy reason it's RGB and not a hex for consistency?

**Comment by warrenbuckley** [comment]
Code changes seem fine - off to test :)

**Comment by bjarnef** [comment]
@nielslyngsoe it seems this affect that `select` now doesn't have outline, when e.g. using keyboard to focus on different elements for exmple in language edit view.

File: src/Umbraco.Web.UI.Client/src/less/forms.less [hunk 2]
```diff
@@ -297,11 +297,11 @@ select[size] {
...
-select:focus,
-input[type="file"]:focus,
-input[type="radio"]:focus,
-input[type="checkbox"]:focus {
-  .tab-focus();
+select,
+input[type="file"],
+input[type="radio"],
+input[type="checkbox"] {
+  .umb-outline();
```
Comment (bjarnef) [inline src/Umbraco.Web.UI.Client/src/less/forms.less]: @nielslyngsoe this use `umb-outline` class mixin, but this has by default `outline: 0`. At the moment `select` doesn't have outline as previous.

image

Comment (nielslyngsoe) [reply src/Umbraco.Web.UI.Client/src/less/forms.less]: @bjarnef your right, this does not make sense. I'm removing "select" from this css-selector. PR: https://github.com/umbraco/Umbraco-CMS/pull/7788

**Pull request merged**